app/_src/policies/meshmetric.md

```diff
@@ -101,6 +101,13 @@ applications:
     port: 8888 # port on which application is listening
 ```
 
+{% if_version gte:2.7.x %}
+{% warning %}
```

Comment:
should we put this at the top of the paragraph?

Reply:
moved